src/App.js

```diff
@@ -1,25 +1,36 @@
-import logo from "./logo.svg";
-import "./App.css";
-import { useEffect, useState } from "react";
+import './App.css';
+import { useEffect, useState } from 'react';
 
 function App() {
-  const [data, setData] = useState(null)
+  const [data, setData] = useState({});
```

Comment:
since you are only using `results` from `data` i would suggest that you destructure results and only save that information